hal/armv7a/zynq7000/zynq.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system kernel
+ *
+ * HAL internal functions for zynq7000
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Maciej Purski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _HAL_ZYNQ_H_
+#define _HAL_ZYNQ_H_
+
+#include "../../cpu.h"
+
+
+int _zynq_setMIO(unsigned int pin, char disableRcvr, char pullup, char ioType, char speed, char l0, char l1, char l2, char l3, char triEnable);
```

Comment:
Missing `extern`.

Reply:
I know we use `extern` in many places, but not everywhere and also it has no effect here.

Reply:
Take a look on other kernel modules. In the majority of headers there is an `extern` keyword. I think that code should be consistent.